aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/Orchestrator.java

```diff
@@ -115,15 +119,21 @@ public boolean isStarted() {
      * @return A Completable operation to start the sync engine orchestrator
      */
     @NonNull
-    public Completable start() {
-        // Only start if it's stopped.
-        if (!OrchestratorStatus.STOPPED.equals(status.get())) {
```

Comment:
It seems like `OrchestratorStatus` needs some helpers like `isStopped`, `isStarting`, etc.

Reply:
I have an isStarted right now. That being said, I did end up changing how we do this a little bit: rather than check the start and perform action, I'm using the `compareAndSet` method of the `AtomicReference`.